### PR TITLE
Ping Annotations

### PR DESCRIPTION
**Pull request by wlach**
Depends on https://github.com/mozilla/glean-annotations/pull/8

### Pull Request checklist

- [x] The pull request has a descriptive title (and a reference to an issue it
      fixes, if applicable)
- [x] All tests and linter checks are passing
- [x] The pull request is free of merge conflicts

### CODE REVIEW

File: scripts/build-glean-metadata.py [hunk 1]
```diff
@@ -9,7 +9,7 @@
 import stringcase
 
 OUTPUT_DIRECTORY = os.path.join("public", "data")
-ANNOTATIONS_URL = "https://mozilla.github.io/glean-annotations/api.json"
+ANNOTATIONS_URL = "https://deploy-preview-8--blissful-roentgen-ca62bb.netlify.app/api.json"
```
Comment (wlach) [inline scripts/build-glean-metadata.py:12]: This is just for testing, it should be reverted before this lands.

Comment (acmiyaguchi) [reply scripts/build-glean-metadata.py:12]: Maybe something to to pull from the environment via `.env`, if it's a common enough workflow.

File: scripts/build-glean-metadata.py [hunk 2]
```diff
@@ -139,7 +139,9 @@ def etl_snake_case(line: str) -> str:
                 app_metrics[metric.identifier] = dict(
                     metric.definition,
                     name=metric.identifier,
-                    annotation=annotations_index.get(app_name, {}).get(metric.identifier),
+                    annotation=annotations_index.get(app_name, {})
+                    .get("metrics", {})
```
Comment (acmiyaguchi) [inline scripts/build-glean-metadata.py:143]: This is black's behavior on long chainable arguments? Imo not the prettiest look (just throwing my opinion out there).

**Review by acmiyaguchi — APPROVED** [review]
Sorry I didn't see this alongside the other PR. LGTM

File: scripts/build-glean-metadata.py (hunk 2 above)
Comment (wlach) [reply scripts/build-glean-metadata.py:143]: Agreed, it's not great. Maybe we can find a better way of expressing this in the future.

File: scripts/build-glean-metadata.py (hunk 1 above)
Comment (wlach) [reply scripts/build-glean-metadata.py:12]: I'm not sure -- in this case I did it so people could preview the behaviour inside the deploy preview here. An environment variable wouldn't solve that case. 

It's worth bearing in mind though, I can see this being useful in some other cases. If this comes up again I'll definitely add this feature.

File: scripts/build-glean-metadata.py (hunk 2 above)
Comment (acmiyaguchi) [reply scripts/build-glean-metadata.py:143]: Maybe wrapping the whole expression in another set of parenthesis might add an indent. Nothing worth nitpicking over though.

Comment (wlach) [reply scripts/build-glean-metadata.py]: Yeah, actually this helped a bunch! I was in the file anyway to revert the change for the annotations URL so it was no trouble.

**Pull request merged**